engine/flink/tests/src/test/resources/extractedTypes/defaultModel.json

```diff
@@ -13128,7 +13128,7 @@
       ],
       "merge": [
         {
-          "description": "Merges two maps. Values in the first map will be overwritten with values from the second map if keys are the same",
+          "description": "Merges two maps. Values in the first map will be overwritten with values from the second map if keys are the same. The keys for the resulting type definition will be sorted.",
```

Comment:
let's remove this pat of text about sorted keys

Reply:
Thanks!